test/metabase/config_test.clj

```diff
@@ -0,0 +1,17 @@
+(ns metabase.config-test
+  (:require [clojure.test :refer :all]
+            [environ.core :as environ]
+            [metabase.config :as config]))
+
+(deftest config-parsing
+  (testing "takes value of non-empty env var"
+    (with-redefs [environ/env (assoc environ/env :max-session-age "123")]
+      (is (= "123"
+             (config/config-str :max-session-age)))))
+  (testing "falls back to default if env var is nil or an empty string"
```

Comment:
This is really nit-picky, but how about also testing the case where the `:max-session-age` is missing from `environ/env`?

Reply:
that would just evaluate to `nil`, which is already handled... i'm inclined to not include a test for that